src/data/markdown/docs/01 guides/02 Using k6/05 Options.md

```diff
@@ -405,6 +405,26 @@ $ k6 run --include-system-env-vars ~/script.js
 
 </div>
 
+### Insecure Skip TLS Verify
+
+A boolean, true or false. When this option is enabled (set to true), all of the verifications that
+would otherwise be done to establish trust in a server provided TLS certificate will be ignored.
+Available in `k6 run` and `k6 cloud` commands
```

Comment:
Maybe an additional comment that this is only about requests done by the script, it doesn't apply for example for requests of outputs (some them have a different option)